admin_guide/encrypting_data.adoc

```diff
@@ -0,0 +1,264 @@
+[[admin-guide-encrypting-data-rest]]
+= Encrypting Data at Rest
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
```

Comment:
I looked at the 7 files I have open and none uses those fields. Not sure if we use them.